fsharp-backend/src/LibService/Telemetry.fs

```diff
@@ -281,24 +274,17 @@ let configureAspNetCore
           with
           | _ -> ""
 
-
-
         activity
         |> Span.addTags [ "meta.type", "http_request"
+                          "http.request.cookies.count", httpRequest.Cookies.Count
+                          "http.request.content_type", httpRequest.ContentType
+                          "http.request.content_length", httpRequest.ContentLength
                           "http.remote_addr", ipAddress
-                          "http.proto", proto
-                          "request.method", httpRequest.Method
-                          "request.path", httpRequest.Path
-                          "request.remote_addr", ipAddress
-                          "request.host", httpRequest.Host
-                          "request.url", httpRequest.GetDisplayUrl()
-                          "request.header.user_agent",
-                          httpRequest.Headers["User-Agent"] ]
```

Comment:
Just to be clear - remote_addr and user_agent are not useful here?

Reply:
The default http opentelemetry already adds `http.user_agent` and `http.remote_addr`